contracts/test/ClaimTest.t.sol

```diff
@@ -0,0 +1,534 @@
+        assertEq(usdc.balanceOf(MAPLE_TREASURY), usdc_treasuryBal        += 0);
+    }
+
+    function setUp() external {
+        /*******************************/
+        /*** Set up actors and state ***/
+        /*******************************/
+
+        start = block.timestamp;
+
+        // Set existing Orthogonal PD as Governor
+        hevm.store(MAPLE_GLOBALS, bytes32(uint256(1)), bytes32(uint256(uint160(address(this)))));
+
+        borrower = new Borrower();
+
+        /*********************************************/
+        /*** Whitelist collateral and funds assets ***/
+        /*********************************************/
+
+        globals.setCollateralAsset(WBTC, true);
+        globals.setLiquidityAsset(USDC, true);
+
+        /*************************************************************/
+        /*** Deploy and set up new LoanFactory with implementation ***/
+        /*************************************************************/
+
+        // Deploy new LoanFactory, implementation, and initializer
+        loanFactory        = new MapleLoanFactory(MAPLE_GLOBALS);
+        loanImplementation = new MapleLoan();
+        loanInitializer    = new MapleLoanInitializer();
+
+        // // Register the new implementations and set default version
+        loanFactory.registerImplementation(1, address(loanImplementation), address(loanInitializer));
+        loanFactory.setDefaultVersion(1);
+
+        globals.setValidLoanFactory(address(loanFactory), true);  // Whitelist new LoanFactory
+
+        /***********************************************/
+        /*** Deploy and set up new DebtLockerFactory ***/
+        /***********************************************/
+
+        // Deploy new LoanFactory, implementation, and initializer
+        debtLockerFactory        = new DebtLockerFactory(MAPLE_GLOBALS);
+        debtLockerImplementation = new DebtLocker();
+        debtLockerInitializer    = new DebtLockerInitializer();
+
+        // Register the new implementations and set default version
+        debtLockerFactory.registerImplementation(1, address(debtLockerImplementation), address(debtLockerInitializer));
+        debtLockerFactory.setDefaultVersion(1);
+
+        globals.setValidSubFactory(POOL_FACTORY, address(debtLockerFactory), true);  // Whitelist new DebtLockerFactory
+    }
+
+    function test_claim_onTimeInterestOnly() external {
+        _createLoan(uint256(1_000_000 * USD), uint256(1_000_000 * USD));
+        _fundLoan();
+        _drawdownLoan();
+
+        // Make a single on time payment
+        ( uint256 principalPortion, uint256 interestPortion ) = _makeLoanPayments(1, false);
+
+        uint256[7] memory details = pool.claim(address(loanV2), address(debtLockerFactory));
+
+        assertEq(usdc.balanceOf(address(loanV2)), 0);       
+
+        assertEq(details[0], 986_301_3698);
+        assertEq(details[1], 986_301_3698);
```

Comment:
USDC has six decimals, so the way to go is take the numbers
`9863013698`, move six in to capture decimals, then do every three (right to left)

`9_863_013698 == $9,863.013698`